frontend/src/member/AddMemberForm.tsx

```diff
@@ -29,6 +29,10 @@ export default function AddMemberForm() {
             .then(() => setTimeout(() => setBackHome(), 2000))
     }
 
+    const setBackHome = () => {
+        navigate("/members/list")
+    }
+
     const [messageStatus, setMessageStatus] = useState('')
```

Comment:
Consider using double quotes instead of single quotes for the sake of consistency